src/main/app/src/app/shared/progress-dialog/progress-dialog.component.html

```diff
@@ -0,0 +1,10 @@
+<!--
+  ~ SPDX-FileCopyrightText: 2021 Atos
```

Comment:
Change to `2024 Lifely` on new files (could we create a lint rule for this?)

Reply:
It needs to be changed to:

```SPDX-FileCopyrightText: 2021 Atos, 2024 Lifely```

Make sure _not_ to remove Atos if we modify existing Atos code! We must do this according to the EUPL or we could run into legal trouble.

Reply:
```suggestion
  ~ SPDX-FileCopyrightText: 2021 Atos, 2024 Lifely
```

Reply:
It is also common best practise. Always mention the original author when modifying existing source code.

Reply:
ps: we had Spotless rules for this but we got of rid of them because it got way too complex to get it working in all circumstance and became unmanageable

Reply:
fixed

Reply:
changed to `SPDX-FileCopyrightText: 2021 Atos, 2024 Lifely`

Reply:
FYI; the ProgressDialogComponent is newly created and so fully 2024 Lifely